api/v1alpha1/types_etcd.go

```diff
@@ -443,7 +446,7 @@ func (e *Etcd) GetServiceAccountName() string {
 
 // GetConfigmapName returns the name of the configmap for the Etcd.
 func (e *Etcd) GetConfigmapName() string {
-	return fmt.Sprintf("etcd-bootstrap-%s", string(e.UID[:6]))
+	return fmt.Sprintf("%s-bootstrap-%s", e.Name, (e.UID[:6]))
```

Comment:
Maybe we should just name this as `<etcd-name>-config-<etcd UID suffix>`. There is no need of including `bootstrap` as part of the name IMHO.

Reply:
The reason I left it like that is because the closed [PR](https://github.com/gardener/etcd-druid/pull/467) tried to do the same and since this is a revival of it, I went with what they proposed in it. But yeah that is good, I'll update it.